commons/types/Availability.d.ts

```diff
@@ -5,16 +5,21 @@ declare namespace Availability {
     slot_size: 15 | 30 | 60;
     start_date: Date;
     dates_to_show: number;
-    available_times: TimeSlot[];
     show_ticks: boolean;
+    calendars: Calendar[];
   }
 
+  interface Calendar {
+    email_address: string;
+    availability: "free" | "busy";
```

Comment:
Can this just be a `boolean` instead? eg `isAvailable`

Reply:
What if we have another status for "partial" in future? I think this gives us a way to be future proof. wdyt?

Reply:
Can we use the same availability enum here and for the Timeslot availability then? It feels a bit strange to have two different sets of definitions for what availability is.

Reply:
I agree. Our APIs use `free | busy` terminology. May be we should change the enum Availability to free | busy | partial

Reply:
opting to use the same enum; didn't want to use boolean because our current endpoints explicitly return a `status: "free" | "busy"` per event right now, rather than a boolean.